### ISSUE TITLE
Missing start.sh

### ISSUE THREAD

**Issue by alxhoff** [felixreimann/opt4j#51]
I am by all means a Java noob and am trying to get the application going. 

I have tried running with gradle `./gradlew run` and then with the executable from sourceforge zip

> Starting Opt4J 3.1.4 (Build 2015-11-28)

Both return the errors below when trying to run the hello world tutorial. Also not sure if it is meant to look like this but the hello world tab is empty. I am sure I am missing some crucial step or something. Maybe detailed in the missing `start.sh` mentioned in the README.

Cheers

> 1) No implementation for java.lang.Integer annotated with @org.opt4j.core.optimizer.MaxIterations() was bound.
>   while locating java.lang.Integer annotated with @org.opt4j.core.optimizer.MaxIterations()
>     for parameter 0 at org.opt4j.core.optimizer.Iteration.<init>(Iteration.java:48)
>   while locating org.opt4j.core.optimizer.Iteration
>     for parameter 5 at org.opt4j.core.optimizer.OptimizationMediator.<init>(OptimizationMediator.java:60)
>   while locating org.opt4j.core.optimizer.Optimizer
> 
> 2) No implementation for org.opt4j.core.optimizer.IterativeOptimizer was bound.
>   while locating org.opt4j.core.optimizer.IterativeOptimizer
>     for parameter 0 at org.opt4j.core.optimizer.OptimizationMediator.<init>(OptimizationMediator.java:60)
>   while locating org.opt4j.core.optimizer.Optimizer
> 
> 2 errors
> 	at org.opt4j.core.start.Opt4JTask.check(Opt4JTask.java:216)
> 	at org.opt4j.core.start.Opt4JTask.execute(Opt4JTask.java:91)
> 	at org.opt4j.core.config.Task.call(Task.java:124)
> 	at org.opt4j.core.config.Task.call(Task.java:42)
> 	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
> 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
> 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
> 	at java.lang.Thread.run(Thread.java:748)
> Caused by: com.google.inject.ConfigurationException: Guice configuration errors:
> 
> 1) No implementation for java.lang.Integer annotated with @org.opt4j.core.optimizer.MaxIterations() was bound.
>   while locating java.lang.Integer annotated with @org.opt4j.core.optimizer.MaxIterations()
>     for parameter 0 at org.opt4j.core.optimizer.Iteration.<init>(Iteration.java:48)
>   while locating org.opt4j.core.optimizer.Iteration
>     for parameter 5 at org.opt4j.core.optimizer.OptimizationMediator.<init>(OptimizationMediator.java:60)
>   while locating org.opt4j.core.optimizer.Optimizer
> 
> 2) No implementation for org.opt4j.core.optimizer.IterativeOptimizer was bound.
>   while locating org.opt4j.core.optimizer.IterativeOptimizer
>     for parameter 0 at org.opt4j.core.optimizer.OptimizationMediator.<init>(OptimizationMediator.java:60)
>   while locating org.opt4j.core.optimizer.Optimizer
> 
> 2 errors
> 	at com.google.inject.internal.InjectorImpl.getProvider(InjectorImpl.java:1035)
> 	at com.google.inject.internal.InjectorImpl.getProvider(InjectorImpl.java:994)
> 	at com.google.inject.internal.InjectorImpl.getInstance(InjectorImpl.java:1044)
> 	at org.opt4j.core.start.Opt4JTask.check(Opt4JTask.java:214)
> 	... 7 more
>

**Comment by FedorSmirnov89** [comment]
Hi. On the first glance, I would say you have an incomplete configuration (we really should improve this section of the tutorial). To run the tutorial application, you need the following modules (the boxes you can klick add or remove in the Opt4J GUI): HelloWorld and EvolutionaryAlgorithm. You probably also want the Viewer so you can see the exploration. Your current error tells you that you don't have an optimizer in the configuration (that's what you need the EA for). The exception is thrown by Guice, so it's a little cryptic.

Long story short: adding the Evolutionary Algorithm and the Viewer module should solve the problem

**Comment by alxhoff** [comment]
Ahhh I didn't know I had to open them as tabs. I was able to run hello world after opening it and the evolutionalry algorithm tab (should anyone in the future have this problem).

Thanks for the quick reply @FedorSmirnov89  :)
